health.go

```diff
@@ -790,6 +790,19 @@ type MinioInfo struct {
 	Services     Services     `json:"services,omitempty"`
 	Backend      interface{}  `json:"backend,omitempty"`
 	Servers      []ServerInfo `json:"servers,omitempty"`
+	TLS          TLSInfo      `json:"tls,omitempty"`
```

Comment:
Omitempty is pointless, since it is not a pointer.

```suggestion
	TLS          TLSInfo      `json:"tls"`
```